commands/new_function.go

```diff
@@ -346,3 +351,33 @@ Cannot have duplicate function names in same yaml file`, functionName, appendFil
 
 	return nil
 }
+
+func addLastNewline(fileName string) error {
```

Comment:
This seems to trying to optimise the problem by reading the last couple of bytes.

Perhaps it could be more paintable and much easier to read if the code read the whole file into memory and used string.HasSuffix()?

Out of interest, where did you source this technique, or did you come up with it yourself?

Reply:
I came up with it on my own. :shrug: Why? I figured it would be fast, regardless of file size. I'm not sure how large stack.yml files get in the real world.

I hadn't considered reading the whole file into a string and then using string.HasSuffix(). Thank you for the suggestion, I'll give that a whirl.

Reply:
Resolved in 3a2cd9c